constants/constants.ts

```diff
@@ -67,7 +67,7 @@ export const SUSHI_OWNER_ADDRESS = {
 };
 
 export const Y_WETH_ADDRESS = "0xa9fE4601811213c340e850ea305481afF02f5b28";
-export const Y_USDC_ADDRESS = "0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9";
+export const YVUSDC_V0_4_3 = "0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE";
```

Comment:
Why is it version `V0_4_3`?

Reply:
Adding the yearn's versioning for yvUSDC. 0.4.3 is the new version.